frontend/src/d3/GraphSimulation.ts

```diff
@@ -180,17 +211,51 @@ export default class {
       .data(this.edges)
       .attr(
         'transform',
-        (link: any) =>
-          `translate(${(link.source.x + link.target.x) / 2},${
-            (link.source.y + link.target.y) / 2
+        (edge: any) =>
+          `translate(${(edge.source.x + edge.target.x) / 2},${
+            (edge.source.y + edge.target.y) / 2
           })`,
       );
 
+    /*
     g.selectChild(this.selectNodeOrEdge)
-      .attr('x1', (link: any) => link.source.x - (link.source.x + link.target.x) / 2)
-      .attr('y1', (link: any) => link.source.y - (link.source.y + link.target.y) / 2)
-      .attr('x2', (link: any) => link.target.x - (link.source.x + link.target.x) / 2)
-      .attr('y2', (link: any) => link.target.y - (link.source.y + link.target.y) / 2);
+      .attr('x1', (edge: any) => edge.source.x - (edge.source.x + edge.target.x) / 2)
+      .attr('y1', (edge: any) => edge.source.y - (edge.source.y + edge.target.y) / 2)
+      .attr('x2', (edge: any) => edge.target.x - (edge.source.x + edge.target.x) / 2)
+      .attr('y2', (edge: any) => edge.target.y - (edge.source.y + edge.target.y) / 2);
+*/
+    // Dette er et forslag som jeg tror kan spare litt scripting tid.
```

Comment:
Vet ikke helt om det sparer tid? Blir vel heller mer tid fordi vi må kjøre en ekstra select(this).

Reply:
ja, vet. Men tanken var at man slipper unna de 5 arrow funksjonene som iterer gjennom all noder og henter ut 1 verdi hver. Er sikkert ikke så farlig :D